### PR TITLE
Created czech.yml

### PR DESCRIPTION
**Pull request by Shadowhackercz**
I have added czech translations

### CODE REVIEW

File: src/main/resources/languages/czech.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  unloaded: "<prefix><#55ffa4>Úspěšně odnačten"
+  no_armor_item: "<prefix><#fa4943>Žádný item s názvem <name> nenalezen. Nelze použít <armor_layer_file>"
+  duplicate_armor_color: "<prefix><#fa4943><first_armor_prefix> set armoru používá stejnou barvu jako <second_armor_prefix> s jinou texturou"
+
+command:
+  help: "<prefix><#b8bbc2>Dostupné příkazy
+        
+<hover:show_text:\"<gray>vyplň reload příkaz\"><click:SUGGEST_COMMAND:/oraxen reload ><#6f737d>/o
+          <#b8bbc2>reload <#fa8943>type</click></hover>  <#6f737d>»<#b8bbc2> reload pluginu
+
```
Comment (Boy0000) [inline src/main/resources/languages/czech.yml]: seems command section has some indentation issues
fix these and ill merge it 👍

**Review by Boy0000 — CHANGES_REQUESTED** [review]
just fix those indentations and its good

File: src/main/resources/languages/czech.yml (hunk 1 above)
Comment (Shadowhackercz) [reply src/main/resources/languages/czech.yml]: Sure, I saw it, doing it right now

**Comment by Shadowhackercz** [comment]
I got a file sent from a friend and it got messed up a bit, I hope it's alright now! :) Neat plugin, by the way.

**Review by Boy0000 — APPROVED** [review]

**Comment by Boy0000** [comment]
yeah lgtm, dont speak czech ofc but

**Comment by Shadowhackercz** [comment]
Well, I have some history of translating, so I guess it should be okay 👍
Thanks

**Pull request merged**